renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
```

Comment:
Do you think we can throw in auto merge for GitHub actions here? 
And maybe disable the dependency dashboard?

Reply:
Yeah I'm happy to do both here 👍🏼